packages/concerto-core/test/data/parser/functiondeclaration.bracesmissing.js

```diff
@@ -1,26 +0,0 @@
-/*
```

Comment:
Are those files dead code?

Reply:
Yes, these are legacy from when Composer supported "transaction processor functions". This makes me wonder whether there is other dead code anywhere, maybe even in the parser?